storage/mwm_id.hpp

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "platform/country_file.hpp"
+
+#include "base/assert.hpp"
+#include "base/checked_cast.hpp"
+
+#include <cstdint>
+#include <map>
+#include <vector>
+
+namespace storage
+{
+using NumMwmId = std::uint16_t;
+
+class NumMwmIds final
```

Comment:
I mean that cross-mwm-routing uses all mwms that are on the device, and sometimes it happen that there are two mwms that correspond to the same area (say, Moscow.mwm), but with different versions, so these two mwms may be completely different. They'll have different MwmIds, but according to your code they'll have the same NumMwmId.